bin/cluster.js

```diff
@@ -12,27 +12,44 @@ if (cluster.isMaster) {
       process.exit(1)
     } else {
       debug('Starting cluster with pid: ' + process.pid)
+
+      const updateIntervals = []
+
       // ensure workers exit cleanly
       process.on('SIGINT', () => {
         debug('Cluster shutting down...')
         for (let worker of Object.values(cluster.workers)) {
           worker.kill()
         }
         // exit the master process
+        // updateIntervals.forEach(i => clearInterval(i))
         process.exit(0)
       })
 
       // ensure workers have a valid schema to serve before spawning them
       db.connect(settings.dbsettings).then(() =>
         db.setupSchema()
       ).then(() => {
+        // set database update intervals
+        // spawnCmd('node', [ 'scripts/sync.js', 'index', settings.index.index_mode || 'update' ])
+        // updateIntervals.push(setInterval(function () {
+        //   spawnCmd('node', [ 'scripts/sync.js', 'index', 'update' ])
+        // }, settings.sync_timeout))
+        // updateIntervals.push(setInterval(function () {
+        //   spawnCmd('node', [ 'scripts/sync.js', 'market' ])
+        // }, settings.market_timeout))
+        // updateIntervals.push(setInterval(function () {
+        //   spawnCmd('node', [ 'scripts/peers.js' ])
+        // }, settings.peer_timeout))
+
```

Comment:
What's the plan here?

Reply:
Sorry, I should mention, I'm still debugging a couple database issues (hence the 'WIP') and I just wanted to manually run the indexing myself for testing, this will be uncommented when I finish.